app/Models/H5pElasticsearchField.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class H5pElasticsearchField extends Model
+{
+    /**
+     * The table associated with the model.
+     *
+     * @var string
+     */
+    protected $table = 'h5p_elasticsearch_fields';
+
+    /**
+     * Get the library that contains field.
+     */
+    public function library()
+    {
+        return $this->belongsTo('App\Models\H5pLibrary');
+    }
+}
```

Comment:
Probably it's better to add this model inside H5P.

Reply:
This model is not part of core H5P,
we have created it in studio to help us decide which fields to index,
so I believe it should remain where studio models are,
however I'll update the reference to H5pLibrary in H5P.